doc/user_guide/py_dataframe.md

```diff
@@ -14,7 +14,47 @@ The parameters of `get_dataframe` are the following.
+| object                         | float            | DOUBLE        | all standard and python3       | all         | Uses None or numpy.NaN for NULL   |
+| object                         | decimal.Decimal  | DECIMAL       | all standard and python3       | all         | Uses None for NULL                |
+| object                         | decimal.Decimal  | DOUBLE        | all standard and python3       | all         | Uses None for NULL                |
+| object                         | bool             | BOOLEAN       | all standard and python3       | all         | Uses None for NULL                |
+| object                         | str              | (VAR)CHAR     | all standard and python3       | all         | Uses None for NULL                |
+| object                         | pandas.Timestamp | TIMESTAMP     | all standard and python3       | >=6.1.0     | Uses None or pandas.NaN for NULL  |
+| object                         | datetime.date    | DATE          | all standard and python3       | all         | Uses None for NULL                |
+| (u)int*[pyarrow]               | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| (u)int*[pyarrow]               | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| float*[pyarrow]                | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| float*[pyarrow]                | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| string[pyarrow]                | -                | (VAR)CHAR     | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| bool[pyarrow]                  | -                | BOOLEAN       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| decimal128(*)[pyarrow]         | -                | DECIMAL       | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| decimal128(*)[pyarrow]         | -                | DOUBLE        | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support               |
+| timestamp[ns, tz=UTC][pyarrow] | -                | TIMESTAMP     | standard-8.0.0 and all python3 | >=6.1.0     | Native NULL support.              |
+
+**Note**:
+
+- Before SLC version 6.1.0, emitting float16 lead to a silent data corruption of the emitted values.
+- Since SLC version 6.1.0 DateFrame Columns with DType object also support pandas.NaN for NULL. 
+- We only DType timestamp[ns, tz=UTC][pyarrow] and datetime64[ns], because Exasol doesn't support timezones. We also drop the timezone before using the timestamp in Exasol. Furthermore, Exasol 7.* only supports miliseconds precision timestamps, the nanoseconds will be truncated.
```

Comment:
```suggestion
- We only support DType timestamp[ns, tz=UTC][pyarrow] and datetime64[ns], because Exasol doesn't support timezones. We also drop the timezone before using the timestamp in Exasol. Furthermore, Exasol 7.* only supports miliseconds precision timestamps, the nanoseconds will be truncated.
```